cluster-autoscaler/go.mod-extra

```diff
@@ -3,7 +3,6 @@ go 1.12
 
 require (
   github.com/rancher/go-rancher v0.1.0
-  github.com/google/go-querystring v1.0.0
```

Comment:
I don't see any pkg deletion. Do you needs this change?

Reply:
When I ran `hack/update-vendor.sh`, it said that this was removed. I had to remove it to get the script to finish successfully.